conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,265 @@
+* CCS style resources (to align with docs best practices)
+* Quick start writing template (to maintain consistency)
+* **Team responsible:** Contributing team; UXD team
+* **Time frame:** After conceptual design story (Sprint 2)
+* **Definition of done:**
+  * All steps, descriptions, messaging, and other content is completely written.
+  * A UXD-reviewed draft is ready for CCS review.
+
+#### Phase 3: CCS review
+CCS Technical Writer conducts a review of the draft.
+
+During this step, have a checkpoint/feedback loop with CCS to give UX feedback on existing documentation. A smaller group can work together and identify what it will take to make existing documentation consumable for quick starts.
+
+* **Team responsible:** CCS Technical Writing; UXD Design
+* **Time frame:** Sprint 2
+* **Definition of done:**
+  * CCS Technical Writer reviewed all content and left edits/comments.
+  * UXD Designer addressed all feedback.
+
+#### Phase 4: UXD Content review
+Abi conducts a content review of the draft.
+
+* **Team responsible:** UXD Content Strategy; UXD Design
+* **Time frame:** Sprint 3
+* **Definition of done:**
+  * Abi reviewed all content and left edits/comments.
+  * UXD Designer addressed all feedback.
+  * Stakeholder review (by Product Management and Dev Lead) is complete and all feedback is applied.
+
+#### Phase 5: Implementation
+UXD Developer implements the quick start design and written content in OpenShift.
+
+* **Team responsible:** UXD Design; UXD Development; Product Management
+* **Time frame:** Initial Development Sprint (Sprint 4)
+* **Definition of done:**
+  * Implementation addresses the main goals of the design and quick start draft.
+  * PR is reviewed by UX and PM and approved to be merged.
+
+#### Phase 6: QE
+QE does a review. UXD reviews and files bugs as needed.
+
+* **Team responsible:** QE; UXD Design; UXD Development
+* **Time frame:** Before feature freeze
+* **Definition of done:**
+  * High-severity bugs are resolved.
+  * Low-severity bugs are triaged for following release.
+
+#### Post-release: User testing (if prioritized)
+If we have the time and resources for user testing, UXD Research will conduct testing.
+
+#### Ongoing: Maintenance and localization
+Quick starts are localized as part of [UXD's localization efforts for OpenShift](https://docs.google.com/document/d/1LnussSddUj9NVkQ2nQ61w42Hj9wAkzWEn1_qmyUXbo8/edit).
+
+NOTE: There needs to be a flag and QE process to review and sign off on quick starts for each release to make sure they still work.
+
+### External contributions
+* External teams may contribute quick starts to the console through a CR in 4.7 and later versions of OpenShift
+* The teams should contribute their quick start to the relevant quick start GitHub repo
+* The GitHub should have the quick start template and the required tag to get UX feedback
+  * A template is in progress and will be provided for teams to use to create their own content
```

Comment:
What template are we referring to? I think what we mean is the style guidelines in this doc. 

Also, just a quick nit: Can we add periods after these list items since they're sentences?